### DISCUSSION TITLE
Programming ATtiny24 with UltraSonic sensor

### DISCUSSION THREAD

**Discussion by 5ghub** [SpenceKonde/ATTinyCore#599] [Support / Q & A]
Hell guys,

I have an UltraSonic board based on this schematic here:
https://uglyduck.vajn.icu/ep/archive/2014/01/Making_a_better_HC_SR04_Echo_Locator.html

on the ultrasonic sensor, it uses MOSI/MISO/RST/VCC/GND.      These pins shall be to ECHO and TRIGGER pins for Ultrasonic sensor

I need help how to program ATtiny24 and then how to wire an Arduino sketch to fire trigger and read ECHO.

If you can help on me will be great. I can also provide payment for this help and consultation.

My Email: hossamfattah@gmail.com   and Skype: hossamfattah

**Comment by SpenceKonde** [comment]
good lord, with all the posts in that thread, nobody ever completed and posted code to make the tiny24 do that? Considering that there was such a great deal of interest on that subject (I have seen that page before) that sort of suggests that actually making working firmware is a hell of a lot harder than people have imagined, otherwise someone would have posted f/w for it. I don't have the background in ultrasonics for that sort of thing. I didn't have time to scour it, but if in all that talk of it, NOBODY has posted code, that would be a badsign indeed about how hard a task it is.
